modin/pandas/series.py

```diff
@@ -678,13 +661,24 @@ def f(x):
             else:
                 return result
         else:
-            result = getattr(sys.modules[self.__module__], return_type)(
-                query_compiler=result
-            )
+            result = globals()[return_type](query_compiler=result)
             if result.name == self.index[0]:
                 result.name = None
             return result
 
+    def aggregate(self, func=None, axis=0, *args, **kwargs):
+        def error_raiser(msg, exception):
+            """Convert passed exception to the same type as pandas do and raise it."""
+            # HACK: to concord with pandas error types by replacing all of the
+            # TypeErrors to the AssertionErrors
+            exception = exception if exception is not TypeError else AssertionError
+            raise exception(msg)
+
+        self._validate_function(func, on_invalid=error_raiser)
+        return super(Series, self).aggregate(func, axis, *args, **kwargs)
+
+    agg = aggregate
+
```

Comment:
Why this function was moved to down?

Reply:
for no good reason, might be an artifact of a `git rebase`... let me fix it.